### PR TITLE
xonsh 0.3.2 -> 0.3.4 and pygment

### PR DESCRIPTION
**Pull request by rardiol**
###### Motivation for this change

Update xonsh and make it start without failing due to missing pygment.
###### Things done
- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

maintainers @nckx @spwhitt  @garbas

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -16891,8 +16891,8 @@ in modules // {
       rm prompt_toolkit/win32_types.py
     '';
 
-    buildInputs = with self; [ jedi ipython pygments ];
-    propagatedBuildInputs = with self; [ docopt six wcwidth ];
+    buildInputs = with self; [ jedi ipython ];
+    propagatedBuildInputs = with self; [ docopt six wcwidth pygments ];
```
Comment (nckx) [inline pkgs/top-level/python-packages.nix]: :+1: Thanks!

**Comment by zimbatm** [comment]
It looks like some tests are failing:
```
======================================================================
ERROR: magic (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: magic
Traceback (most recent call last):
  File "/nix/store/w9gwivwr7ih57b4y1dfhv64xy316p5qn-python3-3.5.1/lib/python3.5/unittest/loader.py", line 153, in loadTestsFromName
    module = __import__(module_name)
  File "/tmp/nix-build-python3.5-mycli-1.6.0.drv-0/mycli-v1.6.0-src/mycli/magic.py", line 2, in <module>
    import sql.parse
ImportError: No module named 'sql'


======================================================================
ERROR: ordereddict (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: ordereddict
Traceback (most recent call last):
  File "/nix/store/w9gwivwr7ih57b4y1dfhv64xy316p5qn-python3-3.5.1/lib/python3.5/unittest/loader.py", line 153, in loadTestsFromName
    module = __import__(module_name)
  File "/tmp/nix-build-python3.5-mycli-1.6.0.drv-0/mycli-v1.6.0-src/mycli/packages/ordereddict.py", line 23, in <module>
    from UserDict import DictMixin
ImportError: No module named 'UserDict'
```

**Comment by rardiol** [comment]
Those tests seem to have already been broken before. UserDict isn't even on python3.

**Comment by garbas** [comment]
manually merged [here](https://github.com/NixOS/nixpkgs/compare/453086a15fc0...0880f1f335b1)

**Pull request closed without merging**